sdks/python/apache_beam/runners/dataflow/internal/dependency.py

```diff
@@ -474,10 +474,29 @@ def _stage_beam_sdk_tarball(sdk_remote_location, staged_path, temp_dir):
         'type of location: %s' % sdk_remote_location)
 
 
-def get_required_container_version():
+def get_default_container_image_for_current_sdk(job_type):
   """For internal use only; no backwards-compatibility guarantees.
 
-  Returns the Google Cloud Dataflow container version for remote execution.
+  Args:
+    job_type: string, BEAM job type.
```

Comment:
Please follow this style guide for structured pydocs when specifying argument / return types: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html (see the other possible style guides here: https://stackoverflow.com/questions/34331088/how-to-comment-parameters-for-pydoc)